PluginTemplateProject/pom.xml

```diff
@@ -131,6 +131,11 @@
                 <groupId>org.scala-tools</groupId>
                 <artifactId>maven-scala-plugin</artifactId>
                 <version>2.15.2</version>
+                <configuration>
+                    <excludes>
+                        <exclude>com/alpine/templates/db/*</exclude>
+                    </excludes>
+                </configuration>
```

Comment:
She has to exclude it so that it will compile I presume

On Thu, Mar 3, 2016 at 11:18 AM, Jenny Thompson <notifications@github.com>
wrote:

> In PluginTemplateProject/pom.xml
> <https://github.com/AlpineNow/CustomPlugins/pull/15#discussion_r54931736>:
>
> > @@ -131,6 +131,11 @@
> >                  <groupId>org.scala-tools</groupId>
> >                  <artifactId>maven-scala-plugin</artifactId>
> >                  <version>2.15.2</version>
> > +                <configuration>
> > +                    <excludes>
> > +                        <exclude>com/alpine/templates/db/*</exclude>
> > +                    </excludes>
> > +                </configuration>
>
> What does this mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AlpineNow/CustomPlugins/pull/15/files#r54931736>.
>